config/Betterloadingscreen/betterloadingscreen.cfg

```diff
@@ -32,7 +32,7 @@ general {
     S:font=textures/font/ascii.png
 
     # Play a sound after minecraft has finished starting up [default: true]
-    B:playSound=false
+    B:playSound=true
 
     # What sound to play when loading is complete. Default is the level up sound (betterloadingscreen:rhapsodia_orb) [default: betterloadingscreen:rhapsodia_orb]
     S:sound=betterloadingscreen:rhapsodia_orb
```

Comment:
This is subjective, but I personally find the `S:sound=minecraft:random.orb` to be more pleasant than the default sound effect here.

Reply:
I've edited your pull request to use the discussed sound effect.